### DISCUSSION TITLE
Screen capture via QR Command

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#1091] [General]
Having a look thru some of the control commands at https://gopro.github.io/labs/control/tech/, I can see we have options for frame grabs.

Is there any (undocumented?) command available to do a screen capture and export as MP4?

**Comment by dnewman-gpsw** [comment]
Not as an MP4, that would be cool, only as JPEGs.  This will grab JPEGs until you power off the camera: ```!F!R```

**Reply by dnewman-gpsw** [reply]
goes into MISC

**Reply by RikshaDriver** [reply]
Would be handy to have a video export option. Sounds like a labs feature request.

For the JPG, what's the frequency of captures and is a higher resolution possible?

**Reply by dnewman-gpsw** [reply]
The grab resolution is the display resolution, so no higher is available.  Frequency 3-5Hz.  Labs can't use the hardware H.264/HEVC encoders as that would break regular capture.  So it will not be a feature coming to Labs, as software encoding is too slow.
